.env.sample

```diff
@@ -2,3 +2,6 @@ PRISMA_HOST=
 PRISMA_SECRET=
 GOOGLE_CLIENT_ID=
 NODEMAILER_PASSWORD=
+AWS_IAM_ACCESS_KEY_ID=
+AWS_IAM_SECRET_ACCESS_KEY=
+BUCKET_NAME=
```

Comment:
remove `BUCKET_NAME` as this depends on application

Reply:
for this, should we still store every bucket names in `.env`? something like `<APP-NAME>_BUCKET_NAME`, get the value in `createEvent()` and pass that as the function parameter. I proposed this since bucket name might be a sensitive information?? 😂

Reply:
good point, but what does the public url for accessing images in a bucket look like? the url that an application uses to fetch an image. does that contain some kind of `bucket name` information? if so, there's no point using `env` here

Reply:
yeah it does actually, it contains everything from `bucket name` to `object name`.

Reply:
then there's no point keeping that information in `env`